src/MSBuild/Resources/Strings.resx

```diff
@@ -707,6 +707,21 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
     </comment>
   </data>
+  <data name="HelpMessage_34_InteractiveSwitch" UESanitized="false" Visibility="Public">
+    <value>  /interactive[:True|False]
+                     Indicates that actions in the build are allowed to
+                     interact with the user.  When enabled, the build can
+                     block indefinitely so it should only be specified
+                     in an environment where a user is present.
```

Comment:
Should we invert this guidance to something like

> It should never be specified in a fully automated scenario.

? Feels like a warning on when you shouldn't use it might be better/scarier than guidance on when it should?

Reply:
I really struggled on the wording here.  Should I change it to:

```
Indicates that actions in the build are allowed to
interact with the user.  Do not use this argument
in a fully automated scenario.
```

Reply:
Yeah, I think so. But I'm also struggling with this one.

Reply:
@AndyGerlicher you must decide on the usage text 😄

Reply:
```
Indicates that actions in the build are allowed to
interact with the user.  Do not use this argument
in an automated scenario where interactivity is
not expected.
```

That was my stab at the last part, but I don't know... any of these are fine really :). I would probably get rid of the rest though, it's not more helpful.

Reply:
I like your wording, I've updated it accordingly